extension/src/lttb.rs

```diff
@@ -81,14 +83,14 @@ pub fn lttb_final_inner(
             state.series.sort_by_key(|point| point.ts);
             let series = Cow::from(&state.series);
             let downsampled = lttb(&*series, state.resolution);
-            flatten!(
-                Timevector {
-                    num_points: downsampled.len() as u32,
-                    is_sorted: true,
-                    internal_padding: [0; 3],
-                    points: (&*downsampled).into(),
-                }
-            ).into()
+            flatten!(Timevector {
+                num_points: downsampled.len() as u32,
+                flags: time_vector::FLAG_IS_SORTED,
+                internal_padding: [0; 3],
+                points: (&*downsampled).into(),
+                null_val: std::vec::from_elem(0_u8, (downsampled.len() + 7) / 8).into()
```

Comment:
What _is_ `from_elem()`? I've never heard of the function before, and I can't find any refrences to it in the docs...

If it's what I think it is, it's probably more idiomatic to do
```rust
null_val: vec![0_u8; (downsampled.len() + 7) / 8].into()
```